packages/components/card/src/card.tsx

```diff
@@ -44,23 +64,63 @@ const Card = (props: TCardProps) => (
       background: ${props.theme === 'dark'
         ? designTokens.colorNeutral95
         : designTokens.colorSurface};
-    `}
-    // Allow to override the styles by passing a `className` prop.
-    // Custom styles can also be passed using the `css` prop from emotion.
-    // https://emotion.sh/docs/css-prop#style-precedence
-    className={props.className}
-  >
-    {props.insetScale === 'none' ? (
-      // Use a `<div>` to ensure that there is always a wrapper container.
-      // This is mostly useful in case custom styles are targeting this element.
+      cursor: ${props.isDisabled
+        ? 'not-allowed'
+        : isClickable
+        ? 'pointer'
+        : 'default'};
+      pointer-events: ${props.isDisabled ? 'none' : 'auto'};
```

Comment:
setting `cursor` and `pointer-events` on the same node is mutually exclusive. Are we ok with creating a new `div` container between this one and the content so we can match the design spec (image below)? The tradeoff is, of course, more nodes and a deeper tree.

I can set the cursor here, and attach the pointer events styling and click handler to the first child.

![image](https://github.com/commercetools/ui-kit/assets/15024562/b692026a-63f9-4924-a7b2-c9757b435cb0)

Reply:
What if we set pointer-events directly for the `<a>`?
```
a {
  pointer-events: ${props.isDisabled ? 'none' : 'auto'};
}
```

Reply:
Since the `<a>` tag is possibly the parent node, we will still be unable to view `cursor: not-allowed` styling if `pointer-events: none` is configured

Reply:
@FilPob, would you be ok removing the "cursor: not-allowed" portion of the design to improve the semantics of this component and ease development? I imagine the opacity change would be enough to indicate this to the user, mimicking our existing design on disabled buttons

Reply:
> Since the `<a>` tag is possibly the parent node, we will still be unable to view `cursor: not-allowed` styling if `pointer-events: none` is configured

Understood. Personally, I'm fine with using a conditional wrapper to handle scenarios where the card shouldn't be clickable. I believe this approach should suffice:

```tsx
const DisablingWrapper = (props: TCardProps) => {
  const isNotClickable = props.isDisabled && (props.onClick || props.to);
  return isNotClickable ? (
    <div
      css={css`
        cursor: not-allowed;
        > a,
        div {
          display: block;
          pointer-events: none;
        }
      `}
    >
      {props.children}
    </div>
  ) : (
    <>{props.children}</>
  );
};
```

Reply:
Hmm I dont want to block the progress in order to fulfil this requirement, as the disabled state for a card will not be used very often I suppose. it is definitely rather an edge case. But for the completeness it would be a nice to have. I would like to ask @CarlosCortizasCT if he maybe has an idea, as he originally (and very correctly) made the point about the disabled state... If it turns out to be tricky, or bad practise code-wise, I can imagine to skip this requirement.

Reply:
I personally felt `pointer-events: none` was a more succinct way of disabling event handlers as opposed to using JS to conditionally apply them, but perhaps it might be time to start using more branching to do so 🤔

Reply:
@CarlosCortizasCT @kark See 4b94814bce1a13c7fdd03fb6bae33964a85f2b7c, thanks for questioning the `pointer-events` strategy I had implemented because I feel this gets us where we need to be, and is more readable. Ultimately, if the card isn't clickable, render the `<div>` parent, otherwise we render a `<a>`

Reply:
@CarlosCortizasCT @kark added a test in ac80da0bc214b42a5630cb682b691777e0641953 for completeness

Reply:
Also @FilPob, this change allows us to have the disabled cursor styling - sorry for the wild goose chase here!

Reply:
awesome! :) thanks for the effort